### PR TITLE
Switch to Python3

### PR DESCRIPTION
**Pull request by HannesSommer**

### CODE REVIEW

File: cuckoo_time_translator_python/setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # fetch values from package.xml
 setup_args = generate_distutils_setup(
-    packages=['cuckoo_time_translator_python'],
+    packages=['cuckoo_time_translator_python', 'cuckoo_time_translator_python.timestamp_series'],
```
Comment (huberya) [inline cuckoo_time_translator_python/setup.py:9]: That looks wrong to me
timestamp_series is a file so I dont think it should be in packages
Why do you think thats needed?

File: cuckoo_time_translator_python/python/cuckoo_time_translator_python/device_time_bags.py [hunk 2]
```diff
@@ -4,9 +4,9 @@
 import rosbag
 import numpy as np
 
-from timestamp_series import TimestampSeries
-from timestamp_series import TimestampSeries
-from tools import *
+from .timestamp_series import TimestampSeries
+from .timestamp_series import TimestampSeries
+from .tools import *
```
Comment (huberya) [inline cuckoo_time_translator_python/python/cuckoo_time_translator_python/device_time_bags.py:9]: I think this breaks python2 (If I remember my struggles with . correctly)
If we care about that what should work in both versions is specifying the full path, i.e.
from cuckoo_time_translator_python.timestamp_series import TimestampSeries

(also one import is duplicated)

File: cuckoo_time_translator_python/python/cuckoo_time_translator_python/plotting.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 import matplotlib.pyplot as plt
 
-from tools import *
+from .tools import *
```
Comment (huberya) [inline cuckoo_time_translator_python/python/cuckoo_time_translator_python/plotting.py:11]: same here

**Review by huberya — APPROVED** [review]

File: cuckoo_time_translator_python/python/cuckoo_time_translator_python/device_time_bags.py (hunk 2 above)
Comment (HannesSommer) [reply cuckoo_time_translator_python/python/cuckoo_time_translator_python/device_time_bags.py]: Thanks. To keep things simple I've decided to drop Python2 support. The Python part should not be used much if at all out there and Python2 was faded out quite some time ago now.

File: cuckoo_time_translator_python/setup.py (hunk 1 above)
Comment (HannesSommer) [reply cuckoo_time_translator_python/setup.py]: Mistake. Thanks for spotting!

**Comment by HannesSommer** [comment]
retest this please

**Pull request merged**